### ISSUE TITLE
connect to external network not working

### ISSUE THREAD

**Issue by GeorgHoffmeyer** [chrisgahlert/gradle-dcompose-plugin#24]
I didn't find any way to connect to an external network. Is that feature missing or how can this be done?

**Comment by chrisgahlert** [comment]
I'm afraid this is not (yet) possible... I'll see if I can come up with something...

**Comment by chrisgahlert** [comment]
This feature will be included in the upcoming release. The syntax will look like this:

```gradle
dcompose {
  myApp {
    image = 'nginx'
    portBindings = ['8080:80']

    // in case I don't want the container to connect to the default network:
    networks = []

    // this syntax can be used to connect to external networks:
    externalNetworks = ['my_network1', 'my_network2']
  }
}
```

For more details about the new properties, see the [wiki page](https://github.com/chrisgahlert/gradle-dcompose-plugin/wiki/02.-List-of-service-properties#runtime-container-properties).

**Comment by chrisgahlert** [comment]
This has just been released in the latest version `0.16.0`. :)
